help-jellyfin/index.html

```diff
@@ -0,0 +1,439 @@
+							<div class="button button__accent" id="frontend_button">I'm a JS developer</div>
+							<div class="button button__accent" id="other_button">I'm another developer</div>
+							</div>
+							</p>
+							<script type="text/javascript">
+								document.getElementById("backend_button").onclick = function() {
+									if (document.getElementById("backend").style.display == 'block') {
+										document.getElementById("frontend").style.display = "none";
+										document.getElementById("other").style.display = "none";
+									}
+									if (document.getElementById("backend").style.display == 'none') {
+										document.getElementById("backend").style.display = "block";
+										document.getElementById("frontend").style.display = "none";
+										document.getElementById("other").style.display = "none";
+										document.getElementById("backend").scrollIntoView(true);
+									} else {
+										document.getElementById("backend").style.display = "none";
+									}
+								}
+								document.getElementById("frontend_button").onclick = function() {
+									if (document.getElementById("frontend").style.display == 'block') {
+										document.getElementById("backend").style.display = "none";
+										document.getElementById("other").style.display = "none";
+									}
+									if (document.getElementById("frontend").style.display == 'none') {
+										document.getElementById("frontend").style.display = "block";
+										document.getElementById("backend").style.display = "none";
+										document.getElementById("other").style.display = "none";
+										document.getElementById("frontend").scrollIntoView(true);
+									} else {
+										document.getElementById("frontend").style.display = "none";
+									}
+								}
+								document.getElementById("other_button").onclick = function() {
+									if (document.getElementById("other").style.display == 'block') {
+										document.getElementById("backend").style.display = "none";
+										document.getElementById("frontend").style.display = "none";
+									}
+									if (document.getElementById("other").style.display == 'none') {
+										document.getElementById("other").style.display = "block";
+										document.getElementById("backend").style.display = "none";
+										document.getElementById("frontend").style.display = "none";
+										document.getElementById("other").scrollIntoView(true);
+									} else {
+										document.getElementById("other").style.display = "none";
+									}
+								}
+							</script>
+							<div id="backend" style="display:none;">
+								<p>The main core of Jellyfin as well as its plugins are written in C#. You have a couple options to get started.
+								<div class="buttons" style="text-align:center">
+								<div class="button button__accent" id="b_bugfix_button">Fix some bugs</div>
+								<div class="button button__accent" id="b_feature_button">Implement a new feature</div>
```

Comment:
I wanted to keep it deeper since that's a "feature" thing, chain of thought being "they want to develop a feature, then they go down and see that it's either a plugin or in-core and get the appropriate section".